app/views/users/_form.html.slim

```diff
@@ -13,4 +13,5 @@
 
   .row
     .col-xs-12
-      = f.button :submit, class: 'btn btn-default'
+    = f.button :submit, ( f.object.new_record? ? _('Create User') : _('Update User')),
```

Comment:
I really don't like the buttons this way. If there's no other way then this, i whould recomment to use standard rails translations for those cases.

Maybe it can be done through model attributes http://guides.rubyonrails.org/i18n.html#translations-for-active-record-models